src/main/java/sirius/kernel/xml/AbstractStructuredOutput.java

```diff
@@ -329,4 +331,22 @@ public StructuredOutput nullsafeProperty(@Nonnull String name, @Nullable Object
         property(name, data != null ? data : "");
         return this;
     }
+
+    @Override
+    public StructuredOutput amountProperty(@Nonnull String name,
+                                           @Nullable Amount amount,
+                                           @Nonnull NumberFormat numberFormat,
+                                           boolean smartRound) {
+        if (amount == null || Amount.NOTHING.equals(amount)) {
+            return property(name, null);
+        }
+
+        if (smartRound) {
+            property(name, amount.toSmartRoundedString(numberFormat).getRoundedAmount());
+        } else {
+            property(name, amount.toString(numberFormat).getRoundedAmount());
+        }
```

Comment:
I think we need a proper way to forward this as for JSON this will otherwise add ' around it...